src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = {truck, bulldozer, excavator};
```

Comment:
We introduce variable when:
a) it will be used more than once
b) when expression in not easily readable and by giving proper name to it makes code more clear
c) when expression is too long and we want to split it to several blocks
```suggestion
        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
```
Here is not the case when we obtain any benefit from variables